pyracf/common/security_admin.py

```diff
@@ -683,6 +688,10 @@ def _clean_and_separate(self, value: str) -> Union[list, str]:
         if isinstance(out, list):
             if None in out:
                 return None
+            if "days" in out:
+                del out[out.index("days")]
+                if len(out) == 1:
+                    return out[0]
```

Comment:
What is this change for?

Reply:
In APPC Session, there's a field for the interval where it is written to output as "x days". This allowed the resulting dictionary to have an integer value instead of appearing as [x, 'days'] which is how it came through with the previous logic. We had a similar issue in SETROPTS, but we solved it differently, and the solution couldn't be applied directly here.

Reply:
Can you post a before and after sample of what this looks like in profile extract?

Reply:
The unit testing covers the "After" case:

```
      <message>SESSION INFORMATION</message>
      <message>-------------------</message>
      <message>SESSION KEY IS E3C5E2E3D2C5E800</message>
      <message>SESSION KEY INTERVAL IS 00005 DAYS</message>
      <message>LOCKED</message>
      <message>CONVSEC= CONV</message>
```

becomes:
```
{
    "sessionKeyInterval": 5,
    "locked": True,
    "sessionKey": "e3c5e2e3d2c5e800",
    "securityCheckingLevel": "conv",
}
```

but before this change it was:
```
  {'locked': True,
   'securityCheckingLevel': 'conv',
   'sessionKey': 'e3c5e2e3d2c5e800',
 'sessionKeyInterval': [5, 'days']}
```

Reply:
The associated logic is that the " IS " functionally becomes treated like an "=" where "SESSION KEY INTERVAL" becomes the key and "00005 DAYS" becomes the value. The existing code sees the " " there and breaks the value into a list, which correctly casts "00005" to an int 5, but keeps the string "days" in the list which is no longer necessary because there was no logic here to throw it out. SETROPTS dealt with "days" in a separate function so the solution didn't transfer well.

Reply:
Thank you! I will add a comment about this to the release notes.

Reply:
I don't know if it makes sense to turn the key into `sessionKeyIntervalDays` or if it can be assumed that the user knows that the unit is days?

Reply:
I think I had it set to `sessionKeyVerificationInterval` in `SetroptsAdmin`. I don't know if you think it makes sense to change it to that to make everything consistent? On the other hand we could also update Setropts later to be consistent with General Resource Profile Admin. I don't know what you think?

I also had `sessionKeyVerificationInterval` under a section called `vtam` for `SetroptsAdmin.list_racf_options()` and it may actually be more accurate to put it under a section called `appc`.

This is one of the reasons why I wanted to mark the `SetroptsAdmin.list_racf_options()` function as experimental even though it is more or less complete and working at this point.

Reply:
Yes, I would say that whatever you want to call it, I think living under appc or session would make more sense in the setropts breakdown. I would also say that for both the trait and the extract key, there should be a "max" in there somewhere as the global setting is the maximum value the individual settings can use.

Reply:
So you are thinking that it should be something like `maxSessionKeyVerificationInterval` everywhere basically?

Reply:
personally, in reading both descriptions, I don't think the word verification adds much to this I would be in favor of either `sessionKeyInterval `in resource admin and `maxSessionKeyInterval `in setropts or `sessionKeyVerificationInterval `in resource and `maxSessionKeyVerificationInterval` in setropts.

Reply:
So, if you don't think the word `verification` helps people better understand the purpose of the trait very much, then I am in favor of removing the word `verification`. In that case it sounds like we can leave `sessionKeyInterval` as is in General Resource Admin, but we may want to change `sessionKeyVerificationInterval` in Setropts Admin to `maxSessionKeyInterval` since it is defining the max session key interval allowed for general resource profiles protecting APPC sessions in addition to changing the `vtam` key to `appc` in `SetroptsAdmin.list_racf_options()`.

Reply:
agreed, I can work those changes into this pull request and make any associated documentation changes in the active doc PR for this issue.

Reply:
Sure. if you are able to, I think it would be good to make those changes now instead of adding it to the backlog.

Reply:
Corresponding doc changes made as well.